src/renderer/components/ui/Checkbox/CheckboxWithLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import { Field } from 'formik'
+import { Checkbox } from 'formik-material-ui'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import CheckBoxOutlineBlank from '@material-ui/icons/CheckBoxOutlineBlank'
+import CheckBoxIcon from '@material-ui/icons/CheckBox'
+
+interface CheckboxProps {
+  name: string
+  label: string
+  labelClass: string
+  rootClass: any
+  [s: string]: any
+}
+
+interface StyledCheckboxProps extends CheckboxProps {}
+
+const StyledCheckbox = withStyles({})((props: StyledCheckboxProps) => (
```

Comment:
I would expect you don't need this `withStyles` since you are not passing any styles anyway.